docs/LabVIEW_Debugging_Techniques.md

```diff
@@ -0,0 +1,95 @@
+
+```eval_rst
++-----------------+--------------------------------------------------------------------------+
+|System Channel   |Description                                                               |
++=================+==========================================================================+
+|HP Count         |The number of times the Primary Control Loop reported being late.         |
++-----------------+--------------------------------------------------------------------------+
+|HP Loop Duration |The duration of the Primary Control Loop in nanoseconds.                  |
++-----------------+--------------------------------------------------------------------------+
+|LP Count         |The number of times the Data Processing Loop reported being late.         |
++-----------------+--------------------------------------------------------------------------+
+|Model Count      |The number of times the models have not completed their execution in time.|
++-----------------+--------------------------------------------------------------------------+
+```
+ 
+If the value of the count channels increase over time, the execution host is not achieving the desired loop rates. You can use the system channels in conjunction with an **[alarm](https://www.ni.com/documentation/en/veristand/latest/manual/add-configure-alarm/)** or **[procedure](https://www.ni.com/documentation/en/veristand/latest/manual/add-configure-procedure/)** to handle the event.
+
+#### System Monitor Custom Device
+
+The [System Monitor Custom Device](https://github.com/ni/niveristand-system-monitor-custom-device/releases) tracks memory resources and CPU usage on an RT target running the VeriStand Engine. Set the update rate (Hz) in System Explorer to determine how often the custom device checks CPU and memory usage and sends them to the corresponding channel. The VeriStand System Monitor can only be used on an RT target. The custom device returns an error if you target it to a Windows system.
+
+#### Telemetry Custom Device
+
+The [Telemetry Custom Device](https://github.com/ni/niveristand-telemetry-custom-device/releases) supports VeriStand benchmarking by helping you log system channels and monitor target resources. The usage data is logged to a TDMS file on the target that is running the VeriStand Engine.
+
+#### Real-Time Trace Viewer
+
+VeriStand 2020 provides built-in support for using the **[Real-Time Trace Viewer](https://zone.ni.com/reference/en-XX/help/370715P-01/lvtracehelp/lv_tracetoolkit_help/)** and **[Real-Time Trace Viewer VIs](https://zone.ni.com/reference/en-XX/help/370715P-01/lvtrace/tracetoolkitvis_pal/)** to capture the timing and execution data of VI and thread events for applications running on an RT target. In a LabVIEW VI, select **Tools»Real-Time Module»Trace Viewer** to display the Real-Time Trace Viewer. For more info on how to use Real-Time Trace Viewer, in LabVIEW go to **Help>>LabVIEW Help>>Real-Time Module>>Real-Time Trace Viewer**.
+
+
```

Comment:
In the doc help, there's also a listing for Custom Error Codes.  It is described there, but maybe to be consistent there, this document should have a section that links out to it?